### ISSUE TITLE
iPad 7 blobs are being saved for wrong device.

### ISSUE THREAD

**Issue by Paulshriner1** [airsquared/blobsaver#201]
**Describe the bug**
I have an iPad 7 with identifier iPad7,11 and boardconfig J171AP. When I save blobs using blobsaver, the blobs it produces are for the boardconfig J172AP, effectively making them invalid as it's the wrong device. It happens to me only for this device, as I also have used blobsaver with an iPhone 6s+, iPhone 5S, iPad Mini 2, iPhone SE 1st gen, and iPod Touch 6, and the blobs that blobsaver produces are for the right identifier and boardconfig. For the iPad 7 I made sure to select the correct device, and I also tried specifying the identifier, but still got blobs for the wrong boardconfig. 

**Screenshots**
I don't have any screenshots as blobsaver does not produce any errors.

**Desktop:**
 - OS Version: Windows 10
 - iOS Device: iPad 7 on iOS 13.7

**Checklist (place an x between the brackets to mark as completed):**
 - [x] I have made sure I am on the latest version of blobsaver
 - [x] I have checked the blobsaver wiki to see if there is already a solution to my problem
 - [x] I have tried common troubleshooting tactics such as restarting my computer and reinstalling blobsaver

**Log(s)**
```
tsschecker = C:\Program Files\blobsaver\lib\tsschecker.exe
Extracting build manifest from http://updates-http.cdn-apple.com/2020SummerFCS/fullrestores/001-42522/29BF3875-7696-4B5A-96F0-66C655C7907F/iPad_Educational_13.7_17H35_Restore.ipsw
Extracted to C:\Users\PAULSH~1\AppData\Local\Temp\BuildManifest2635166182174208323.plist
Running: [C:\Program Files\blobsaver\lib\tsschecker.exe, --nocache, -d, iPad7,11, -s, -e, 12345678901234, --save-path, C:\Users\Paul Shriner\Blobs, --generator, 0x1111111111111111, -m, C:\Users\PAULSH~1\AppData\Local\Temp\BuildManifest2635166182174208323.plist]
Version: 55e5c5054699bd3433cc5548c3a0d740b59bc6d6 - 1
libfragmentzip version: 0.1-120447d0f410dffb49948fa155467fc5d91ca3c8
[TSSC] manually specified generator "0x1111111111111111"
[TSSC] manually specified ECID to use, parsed "12345678901234" to dec:12345678901234 hex:b3a73ce2ff2
[TSSC] opening C:\Users\PAULSH~1\AppData\Local\Temp\BuildManifest2635166182174208323.plist
[TSSR] LOG: device iPad7,11 doesn't need a baseband ticket, continuing without requesting a Baseband ticket
[TSSR] Request URL set to https://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 1... success
also requesting APTicket for installType=Update
[TSSR] LOG: device iPad7,11 doesn't need a baseband ticket, continuing without requesting a Baseband ticket
[TSSR] Request URL set to https://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 1... success
[TSSR] User specified doesn't to request a baseband ticket.
[TSSR] Request URL set to https://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 1... success
Saved shsh blobs!

```

**Comment by airsquared** [comment]
This has been fixed in v2.5.5.

**Comment by Paulshriner1** [comment]
I updated to 2.5.5 and can confirm it is saving blobs for my iPad 7 with the correct boardconfig. Thank you so much!
